pkg/config/config.go

```diff
@@ -497,6 +497,7 @@ func InitConfig(config Config) {
 	// contexts will be deleted (see 'dogstatsd_expiry_seconds').
 	config.BindEnvAndSetDefault("dogstatsd_context_expiry_seconds", 300)
 	config.BindEnvAndSetDefault("dogstatsd_origin_detection", false) // Only supported for socket traffic
+	config.BindEnvAndSetDefault("dogstatsd_origin_detection_udp", false)
```

Comment:
not totally convinced by this config name, suggestions are welcome!

Reply:
The setting would also enable the new field on UDS traffic. I think we want that, but the name should probably reflect this. `dogstatsd_origin_detection_full`, or perhaps `dogstatsd_origin_detection: client`.

Reply:
I'd say `dogstatsd_origin_detection_type: client`